cpu/cc26xx_cc13xx/include/cc26xx_cc13xx_rfc_mailbox.h

```diff
@@ -0,0 +1,233 @@
+                                         issued in a context where it is not
+                                         supported */
+    RFC_CMDSTA_SCHEDULINGERROR = 0x86, /**< A radio operation command was
+                                            attempted to be scheduled while
+                                            another operation was already
+                                            running in the RF core */
+    RFC_CMDSTA_PARERROR = 0x87, /**< There were errors in the command parameters
+                                     that are parsed on submission. */
+    RFC_CMDSTA_QUEUEERROR = 0x88, /**< An operation on a data entry queue was
+                                       attempted that was not supported by the
+                                       queue in its current state */
+    RFC_CMDSTA_QUEUEBUSY = 0x89, /**< An operation on a data entry was attempted
+                                      while that entry was busy */
+};
+
+/**
+ * @brief   Macros for sending direct commands
+ * @{
+ */
+/**
+ * @brief   Direct command with no parameter
+ */
+#define RFC_CMDR_DIR_CMD(cmdid) (((cmdid) << 16) | 1)
+
+/**
+ * @brief   Direct command with 1-byte parameter
+ */
+#define RFC_CMDR_DIR_CMD_1BYTE(cmdid, par) (((cmdid) << 16) | ((par) << 8) | 1)
+
+/**
+ * @brief   Direct command with 2-byte parameter
+ */
+#define RFC_CMDR_DIR_CMD_2BYTE(cmdid, par) \
+    (((cmdid) << 16) | ((par) & 0xFFFC) | 1)
+/** @} */
+
+/**
+ * @enum    Definitions for trigger types
+ */
+enum {
+    RFC_TRIG_NOW = 0, /**< Triggers immediately */
+    RFC_TRIG_NEVER = 1, /**< Never trigs */
+    RFC_TRIG_ABSTIME = 2, /**< Trigs at an absolute time */
+    RFC_TRIG_REL_SUBMIT = 3, /**< Trigs at a time relative to the command was
+                                  submitted */
+    RFC_TRIG_REL_START = 4, /**< Trigs at a time relative to the command
+                                 started */
+    RFC_TRIG_REL_PREVSTART = 5, /**< Trigs at a time relative to the previous
+                                     command in the chain started */
+    RFC_TRIG_REL_FIRSTSTART = 6, /**< Trigs at a time relative to the first
+                                      command in the chain started */
+    RFC_TRIG_REL_PREVEND = 7, /**< Trigs at a time relative to the previous
+                                   command in the chain ended */
+    RFC_TRIG_REL_EVT1 = 8, /**< Trigs at a time relative to the context defined
+                                "Event 1" */
+    RFC_TRIG_REL_EVT2 = 9, /**< Trigs at a time relative to the context defined
+                                "Event 2" */
+    RFC_TRIG_EXTERNAL = 10, /**< Trigs at an external event to the radio
+                                 timer */
+    RFC_TRIG_PAST_BM = 0x80, /**< Bitmask for setting past trig bit in order to
+                                  trig immediately if trigger happened in the
+                                  past */
+};
```

Comment:
Minor nit: please align the comments :wink:

Reply:
Aligned!